src/main/java/hudson/plugins/git/GitStatus.java

```diff
@@ -280,6 +280,17 @@ public void writeBody(PrintWriter w) {
                             urlFound = true;
 
                             if (!(project instanceof AbstractProject && ((AbstractProject) project).isDisabled())) {
+                                //JENKINS-30178 Add default parameters defined in the job
+                                List<JobProperty> jobProperties = ((Job) project).getAllProperties();
+                                for (JobProperty jobProperty : jobProperties) {
+                                    if(jobProperty instanceof ParametersDefinitionProperty) {
+                                        ParametersDefinitionProperty parametersDefinitionProperty = (ParametersDefinitionProperty) jobProperty;
+                                        for (ParameterDefinition parameterDefinition : parametersDefinitionProperty.getParameterDefinitions()) {
+                                            parameterDefinition.getDefaultParameterValue();
+                                            buildParameters.add(parameterDefinition.getDefaultParameterValue());
```

Comment:
<code> getDefaultParameterValue()</code> may return null. In such case there may be an NPE in the build